huaweicloud/services/lts/resource_huaweicloud_lts_cross_account_access.go

```diff
@@ -0,0 +1,292 @@
+
+	listOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+
+	name := d.Get("name").(string)
+	listOpt.JSONBody = map[string]interface{}{
+		"access_config_name_list": []string{name},
+	}
+	listResp, err := ltsClient.Request("POST", listPath, &listOpt)
+	if err != nil {
+		return common.CheckDeletedDiag(d, err, "error retrieving LTS cross account access")
+	}
+
+	listRespBody, err := utils.FlattenResponse(listResp)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	jsonPath := fmt.Sprintf("result[?access_config_name=='%s']|[0]", name)
+	listRespBody = utils.PathSearch(jsonPath, listRespBody, nil)
+	if listRespBody == nil {
+		return common.CheckDeletedDiag(d, golangsdk.ErrDefault404{}, "no data found")
+	}
+
+	// update the resource ID for import scenario
+	configID := utils.PathSearch("access_config_id", listRespBody, "")
+	if configID != "" {
+		d.SetId(configID.(string))
+	}
+
+	created := utils.PathSearch("create_time", listRespBody, float64(0)).(float64)
+	mErr := multierror.Append(nil,
+		d.Set("region", region),
+		d.Set("name", utils.PathSearch("access_config_name", listRespBody, nil)),
+		d.Set("created_at", utils.FormatTimeStampRFC3339(int64(created)/1000, false)),
+		d.Set("tags", utils.FlattenTagsToMap(utils.PathSearch("access_config_tag", listRespBody, nil))),
+		d.Set("access_config_type", utils.PathSearch("access_config_type", listRespBody, nil)),
+		d.Set("log_group_id", utils.PathSearch("log_info.log_group_id", listRespBody, nil)),
+		d.Set("log_group_name", utils.PathSearch("log_info.log_group_name", listRespBody, nil)),
+		d.Set("log_stream_id", utils.PathSearch("log_info.log_stream_id", listRespBody, nil)),
+		d.Set("log_stream_name", utils.PathSearch("log_info.log_stream_name", listRespBody, nil)),
+	)
+	return diag.FromErr(mErr.ErrorOrNil())
+}
+
+func resourceCrossAccountAccessUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+	product := "lts"
+	tagsHttpUrl := "v1/{project_id}/{resource_type}/{resource_id}/tags/action"
+
+	ltsClient, err := cfg.NewServiceClient(product, region)
+	if err != nil {
+		return diag.Errorf("error creating LTS client: %s", err)
+	}
+
+	if d.HasChanges("tags") {
+		tagsPath := ltsClient.Endpoint + tagsHttpUrl
+		tagsPath = strings.ReplaceAll(tagsPath, "{project_id}", ltsClient.ProjectID)
+		tagsPath = strings.ReplaceAll(tagsPath, "{resource_type}", "ltsAccessConfig")
+		tagsPath = strings.ReplaceAll(tagsPath, "{resource_id}", d.Id())
+
+		tags := d.Get("tags").(map[string]interface{})
+		tagsOpt := golangsdk.RequestOpts{
+			KeepResponseBody: true,
+		}
+		// When action is delete, it will delete all tags and create new tags as parameter tags
+		tagsOpt.JSONBody = utils.RemoveNil(buildTagsBodyParams(tags, "delete"))
+		_, err = ltsClient.Request("POST", tagsPath, &tagsOpt)
+		if err != nil {
+			return diag.Errorf("error updating cross account access tags: %s", err)
+		}
+	}
```

Comment:
ditto

Reply:
the public function is not support service type of LTS, and the API is not work as other tags API.